benchopt/utils/files.py

```diff
@@ -10,3 +11,21 @@ def rm_folder(folder):
         else:
             f.unlink()
     folder.rmdir()
+
+
+def uniquify_results(file_path):
+    "Add a number to filename if it already exists"
+    parent = file_path.parent
```

Comment:
really minor: from a conceptual point of view, don't these belong more L25 ? They are not needed if file_path.exists, I'd define them as close as possible to where they're used

Reply:
Yes, changed in the latest commit thanks